addon/serializers/echonest.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const { isPresent } = Ember;
+const { RESTSerializer } = DS;
+
+export default RESTSerializer.extend({
+    modelKey: '',
+
+    pluralizeKey(key) {
+        return Ember.String.pluralize(key);
+    },
+
+    normalizeResponse(store, primaryModelClass, payload, id, requestType) {
+        const key = this.get('modelKey');
+        const pluralKey = this.pluralizeKey(key);
+        if (isPresent(key) && isPresent(payload.response) && isPresent(payload.response[pluralKey])) {
+            const items = payload.response[pluralKey].map(function (item, index) {
```

Comment:
`=>` should only be used when you need to bind the lexical `this` to that scope. I'm not referencing `this` anywhere inside the method, so I should use `function`.